src/template_storage.mligo

```diff
@@ -0,0 +1,10 @@
+// SPDX-FileCopyrightText: 2021 TQ Tezos
+// SPDX-License-Identifier: LicenseRef-MIT-TQ
+
+// This type should define the type of the contract-extra field in storage
+// for this variant.
+type contract_extra = (string, bytes) big_map
+
```

Comment:
This commit (ddd9fa692a50961dbd11b12b8aaa819aace33c40) also doesn't have a description in its message and I believe that changes to this file are misplaced and actually belong to a different commit.

Reply:
Have expanded the commit message.